apps/dashboard/app/views/batch_connect/sessions/_panel.html.erb

```diff
@@ -5,7 +5,7 @@
         if session.view
           views = { partial: "custom", locals: { view: session.view, connect: session.connect } }
         else
-          if session.script_type == "vnc"
+          if session.script_type == "vnc" || session.script_type == "vnc_container"
```

Comment:
This is fine, (and thank you again), but I was thinking about adding a `vnc?` API to `BatchConnect:Session`

```ruby
def vnc?
  script_type == "vnc" || script_type == "vnc_container"
end

Reply:
Ah! Yes, that's a more elegant solution. Would that then simplify to:

```ruby
if session.vnc
    ...
```

I'm definitely no ruby expert.

Reply:
Yes but the `?` is important. Not functionally, but it's common a Ruby thing for boolean functions to be questions like that for readability (as opposed to a java-esque `isVnc` or similar).